services/payments/controllers.go

```diff
@@ -85,21 +86,14 @@ func PrepareHandler(service *Service) handlers.AppHandler {
 			return handlers.WrapError(err, "failed to validate transaction", http.StatusBadRequest)
 		}
 
-		// sign the transaction
-		req.PublicKey, req.Signature, err = req.SignTransaction(ctx)
-		if err != nil {
-			logger.Error().Err(err).Str("request", fmt.Sprintf("%+v", req)).Msg("failed to sign transaction")
-			return handlers.WrapError(err, "failed to sign transaction", http.StatusBadRequest)
-		}
-
-		logger.Debug().Str("transaction", fmt.Sprintf("%+v", req)).Msg("handling prepare request")
-
 		// returns an enriched list of transactions, which includes the document metadata
 		resp, err := service.InsertTransaction(ctx, req)
```

Comment:
Should we rename this PrepareTransaction in the service method to match?  Since we are doing everything in the InsertTransaction (instead of signing it outside like we were)

Reply:
I went ahead and did this. I think it may change a bit when I get to ev's feedback, but if not I've gone ahead and resolved. :)